sdpb/api/stations.py

```diff
@@ -287,7 +299,90 @@ def collection(
 
 
 ####
-# /stations/{station_id}/observations/{var_id}
+# /stations/{station_id}/variables/{var_id}
+####
```

Comment:
Would it be helpful to put the station-variables api into a separate module, say `station_variables.py`? It's okay here but it makes for a very long module.